docs/reference/ingest/apis/put-pipeline.asciidoc

```diff
@@ -43,7 +43,13 @@ PUT _ingest/pipeline/my-pipeline-id
 
 `<pipeline>`::
 (Required, string) ID of the ingest pipeline to create or update.
-
++
+[IMPORTANT]
+====
+To avoid naming collisions with built-in and Fleet-managed ingest pipelines, avoid using `@` as part of your own ingest pipelines names.
+The exception of that rule are the `@custom` ingest pipelines that let you safely add a custom pipeline to managed pipelines.
```

Comment:
Perhaps mention that the template is not just named `@custom`?
```suggestion
The exception of that rule are the `<type>@custom` ingest pipelines that let you safely add a custom pipeline to managed pipelines.
```

Reply:
Makes sense. But these custom pipelines are necessarily prefix with just the `data_stream.type`. I've changed to this and other occurrences to `*@custom` now. I think it should be pretty self-explanatory that this means pipelines/component templates whose name end with `@custom`.